package.json

```diff
@@ -27,7 +27,7 @@
     "start:proxy:beta": "NODE_ENV=development BETA=true webpack serve --config config/dev.webpack.config.js",
     "start": "npm run start:proxy",
     "start:beta": "npm run start:proxy:beta",
-    "test": "jest --verbose",
+    "test": "jest --coverage --verbose",
```

Comment:
Shouldn't `"collectCoverage": true,` take care of this without `--coverage`

Reply:
I was about to write the same thing. `jest` entry in package.json should already modify the behavior of jest.

Reply:
Thanks! Updated the PR